frontends/pytorch/python/torch_mlir_utils/codegen/torch_ods_gen.py

```diff
@@ -0,0 +1,481 @@
+#  Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
```

Comment:
Note to self: We should add a Github action which runs this at Torch head and verifies that no diffs were introduced.

Reply:
Good idea. I can take a stab too to get my feet wet with CI stuff too.